.jshintrc

```diff
@@ -2,7 +2,7 @@
     "bitwise": true,
     "eqeqeq": true,
     "curly": true,
-    "strict": true,
+    "strict": false,
```

Comment:
When I use rollup with `--format umd` option the output script starts like the following

```
(function (global, factory) {
	typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory() :
	typeof define === 'function' && define.amd ? define(factory) :
	(global.c3 = factory());
}(this, (function () { 'use strict';
```

And this breaks the `strict` rule of jshint. I couldn't configure rollup to insert `'use strict'` in the first function, so I disable it for the moment.